client/lib/plans/constants.js

```diff
@@ -170,6 +170,14 @@ const WPComGetBillingTimeframe = abtest => {
 	return i18n.translate( 'per month, billed yearly' );
 };
 
+const WPComGetBiennialBillingTimeframe = abtest => {
+	if ( isEnabled( 'upgrades/2-year-plans' ) ) {
+		return i18n.translate( '/month, billed biennially' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **16** times:
`translate( '/month, billed annually or biennially' )` *ES Score: 22*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
Is there a reason for the difference in writing? For yearly, we use `per month` and not `/month`.

Reply:
Well actually it varies. Currently the code looks like this:

```js
const WPComGetBillingTimeframe = abtest => {
	if ( isEnabled( 'upgrades/2-year-plans' ) ) {
		return i18n.translate( '/month, billed annually or biennially' );
	}

	if ( abtest && abtest( 'upgradePricingDisplayV2' ) === 'modified' ) {
		// Note: Don't make this translatable because it's only visible to English-language users
		return '/month, billed annually';
	}
	return i18n.translate( 'per month, billed yearly' );
};
```